src/App.vue

```diff
@@ -151,11 +153,23 @@ export default {
         debouncedEmitUserSessionExpiredError()
       })
     },
+    isAllowedOrigin (origin) {
+      if (!this.originRegExps) {
+        const origins = process.env.VUE_APP_POST_MESSAGE_ALLOWED_ORIGINS
+          .split(',')
+          .map(o => o.trim())
+
+        this.originRegExps = origins.map(o => {
+          const str = `^${escapeRegexp(o).replace(/\\\*/g, '.*')}$`
```

Comment:
Can’t we replace this asterisk before using escapeRexep to avoid escapes?

And I think we are expected to exclude `/` in domain glob matching to avoid including both http and https protocols. We could even skip escape-string-regex this way:

```js
const str = `^${o.replace(/\*/g, '[^/]*')}$`
```

Reply:
I tried to replace the asterisk by `.*` before escaping but it is escaped into `\.\*`.

I'm not sure we can drop protocols in case one wants to specify only HTTPS protocol from a domain.
What I can do is to detect if the configuration origin has a protocol before checking the passed origin.

Reply:
I did not ask to remove protocol. I said leading wildcard includes both protocols which is an issue.

Reply:
You're right. I misread your comment.